src/aws-api-mcp-server/awslabs/aws_api_mcp_server/core/common/__init__.py

```diff
@@ -22,6 +29,40 @@
     Credentials,
     ProgramValidationRequest,
 )
+import sys
+
+
+def initialize_logger():
+    logger.remove()
```

Comment:
👋 . Have you considered this as a more generalized solution that could be used across MCP servers instead of scoping this down to just the API MCP server?